build/buildRunTimeImageBases.sh

```diff
@@ -36,12 +36,19 @@ docker build \
     -t "$RUNTIME_BASE_IMAGE_NAME" \
     $REPO_DIR
 
+if [ "$runtimeSubDir" == "node" ]; then
```

Comment:
These new lines of code are temporary here as I would be sending one more PR (related to pushing runtime bases to MCR) which would have different content.